src/plugins/intel_gpu/src/kernel_selector/kernels/fully_connected/fully_connected_kernel_bf_tiled.cpp

```diff
@@ -385,16 +385,10 @@ KernelsData FullyConnected_bf_tiled::GetTunedKernelsDataByIndex(const Params &pa
 
     tune_params tparams = GetAutoTuneParams(fc_params, autoTuneIndex);
 
-    WeightsLayout weights_layout = WeightsLayout::os_iyx_osv16;
-    if (tparams.tile_ofm * simd == 32)
-        weights_layout = WeightsLayout::os_iyx_osv32;
-    else if (tparams.tile_ofm * simd == 64)
-        weights_layout = WeightsLayout::os_iyx_osv64;
-
     return GetCommonKernelsData(params,
                                 options,
                                 fc_params.inputs[0].GetLayout(),
-                                weights_layout,
+                                WeightsLayout::oiyx,
```

Comment:
Did you have a chance to evaluate performance impact of this change on previous generations as well? Maybe it makes sense to keep weights_layout as a parametrized option under some conditions (e.g. type of GPU or something else)